DNN Platform/Library/Entities/Portals/PortalController.cs

```diff
@@ -826,7 +1121,20 @@ public static string GetPortalDefaultLanguage(int portalID)
         /// <param name = "CultureCode"></param>
         /// <remarks>
         /// </remarks>
-        public static void UpdatePortalDefaultLanguage(int portalID, string CultureCode)
+        [Obsolete("Deprecated in DNN 9.7.0. Use the 'IPortalController.UpdatePortalDefaultLanguage' instnace method. Scheduled removal in v11.0.0.")]
+        public static void UpdatePortalDefaultLanguage(int portalID, string CultureCode) =>
+            UpdatePortalDefaultLanguage(portalID, CultureCode);

```

Comment:
This is just a recursive loop, the static method will keep calling itself.  You'd need to create an `IPortalController` instance to delegate to, if that's your intention.

Reply:
Thanks for catching this, a big mistake if we don't include the keyword `Instance`